Cornelica/scenario/rp01a.txt

```diff
@@ -68,73 +70,74 @@
 
 @se f=愛液1 pitch=50
 
-[リリィ/rry0030]「んふうう……これ……おちんちん……大きい。あっあっ……私の膣中……広げられてくのが分かる。んくっ……はふんんっ」
+[Lily/rry0030]"Fuuu...this...penis.. is large. Ah... i can feel it spreading out...in my vagina. N...argh"
 
-[リリィ/rry0031]「届いてる。アルのおちんちんが奥まで……。子宮に当たってるのが分かる。あはぁあ……凄く熱い……。おまんこの膣中……火傷しちゃいそうなくらい」
+[Lily/rry0031]"It reaches. Al penis is deep down... You know you are hitting the uterus. Ah...It's really hot...My vagina... it's like it burns"
 
-[リリィ/rry0032]「はぁっはぁっはぁっ……どう？　気持ちいい？　私のおまんこ……感じる？」
+[Lily/rry0032]"Hahha... How is it? Does it feel good? My vagina...do you feel it? 
 
-[アル]「ぅあっ！　……ぁ……あああ！」
+[Al]"Ah! ...A..ahhah!
 
-[リリィ/rry0033]「返事をする余裕もないか。んふふ、おちんちんが溶けちゃいそうなくらい感じてるって感じだね。あふぅう……」
+[Lily/rry0033]"You can't even reply. Fufu♪, looks like you feel that the penis is about to melt...Afu ~uu.."
 
 @se f=愛液1 pitch=50
 
-[リリィ/rry0034]「本当に気持ち良さそう。膣中でおちんちんが震えてるのが分かる。ビクビクが伝わって来てるよ。あっは……んはぁあああ……」
+[Lily/rry0034]"It looks really pleasant. I can see that the penis is trembling in the vagina. [?] is coming down. Awwa…… n ha~aha……"
+;;「本当に気持ち良さそう。膣中でおちんちんが震えてるのが分かる。ビクビクが伝わって来てるよ。あっは……んはぁあああ……」
+[Lily/rry0035]"This... just feeling a penis...I...well...Haaaaaaaaaaaaaaa...nice... I feel so much"
 
-[リリィ/rry0035]「これ……おちんちんを感じてるだけで私も……んふう……はぁっはぁっ……いい……すごく感じちゃう」
-
-[リリィ/rry0036]「でも、まだ我慢。アルもまだ我慢だよ。もっと……もっともっと……気持ち良くなるためにね」
+[Lily/rry0036]"But I still have endurance. Al still endurance. More...more, more... to become more pleasant!"
 
 @se f=愛液1 pitch=50
 
 ;;SE■ピストン音
 @flash t=300 wt=0
 @bgv f=パイズリ・手コキ２ pitch=75 wt=0
 
-[リリィ/rry0037]「んっは！　あふあっ！　あっあっあっあっあっ！　あんんんっ！」
+[Lily/rry0037]"Crap! Afua~! Ahahahah! Anth!
 
 @se f=愛液1
 
-　ピストンが始まる。肉壺でペニス全体が、ぐっじゅぐっじゅと擦り上げられる。
+;;ピストンが始まる。肉壺でペニス全体が、ぐっじゅぐっじゅと擦り上げられる。
+ I start to rubbing my penis deep in her vagina
 
-[リリィ/rry0038]「どう？　ヒダヒダで締められるのは……。先っぽを……んんんっ……子宮で吸われるの……すっごくいいでしょ？」
+[Lily/rry0038]"How? You are locked in. Go ahead... you are filling my uterus... Is it really good?"
 
-[リリィ/rry0039]「射精してっておまんこにおねだりされてる気分になっちゃうでしょ？」
+[Lily/rry0039]"How do you feel like being caught in my vagina?"
 
-[リリィ/rry0040]「……でもまだまだだよ。ほら、こうやって……パンパンッて腰を打ち付けたりもしてあげる、んあんっ！　あっふあ……くふんんっ！」
+[Lily/rry0040]"But I have not done it yet. Here... like this...I'll give you a chance to move our hip, ah! Ah ...... Huh!"
 
 @se f=愛液1 pitch=50
 @se f=締め付ける pitch=50
 
 @flash t=300 wt=0
 @bgv f=パイズリ・手コキ２ pitch=90
 
-[リリィ/rry0041]「ギュッギュッておまんこでおちんちん締められるの……感じる？」
+[Lily/rry0041]"I'm tightening your penis in my vagina....Do you feel it?"
 
 @se f=パイズリ・手コキ２ pitch=150
 @se f=締め付ける pitch=50
 @qk
 
-[リリィ/rry0042]「本当に気持ち良さそう。私が……ああっ！　腰を振るたび……おちんちんが大きくなって来てるのが分かる。子宮をズンって圧迫してきてる」
+[Lily/rry0042]"It looks really pleasant. I...aah! Every time I shake my hip... I can feel that your penis is getting bigger. It's pressing against my uterus"
 
-[リリィ/rry0043]「私の膣中がアルのおちんちんの形に変えられていくみたい。ああ……おまんこがドロドロに溶けちゃいそうなくらい……気持ちいい」
+[Lily/rry0043]"It seems that my vagina is being transformed into the form of your penis. Oh... it seems that my vagina will melt in the muddy...it feels good"
```

Comment:
`in the muddy`? Sounds like something got lost in translation I guess. I don't know if `ecstasy` works in this situation.

Reply:
Well turn into mud I think was a... Fuck my english... Like "feeling like a million bucks", a pre-build.